tests/console/validate_repos.pm

```diff
@@ -34,7 +34,7 @@ sub run {
         });
     }
     foreach my $alias (@actual_aliases) {
-        next if ($alias =~ /home_images|home_sles16/);
+        next if ($alias =~ /home_images|home_sles16|15-SP7/);
```

Comment:
please do it via test data if possible, this two other exclusion will disappear when we do not use fork.